### DISCUSSION TITLE
The field 'InvoiceIssuerName' of the record wasn't found.

### DISCUSSION THREAD

**Discussion by ackerk** [microsoft/finops-toolkit#639] [Q&A] [answered]
Hi,

Trying to set up the Power BI CostManagementConnector and data comes in nicely in Power Query, but the CostDetails query returns errors in the 'Output' step because it cannot find the column InvoiceIssuerName.

When looking at the source data and looking for columns with Invoice in their name, I cannot find InvoiceIssuerID or InvoiceIssuerName?!?!?

Anybody any idea?

Thanks in advance!

Wkr,

Kris.

**Comment by flanakin** [comment, marked as answer]
@ackerk What account type do you have? This should be handled already. 

I just did a quick check and I did notice what I believe is a bug that would cause CSP billing accounts/profiles to fail. That's an easy fix that I'll get out in 0.2.1 soon.

The source data uses PartnerTenantId and PartnerName. In the connector report, we transform the data to [FOCUS](https://aka.ms/costmgmt/focus) so we can reuse the logic from the other reports we have.

**Reply by flanakin** [reply]
FYI - This is now available in 0.3. We decided to bump the version given all the things in the release.

**Comment by ackerk** [comment]
@flanakin we have an Azure MCA
